src/services/TextWriter.hh

```diff
@@ -2,7 +2,7 @@
 #define RA_SERVICES_TEXTWRITER
 #pragma once
 
-#include <string>
+#include <xstring>
```

Comment:
xstring is a microsoft specific header, please use the standard string header so we're ready when we eventually get to supporting other platforms.

Reply:
I got you. Non standard stuff should be avoided for easier transition.